### ISSUE TITLE
Number of val batches during training

### ISSUE THREAD

**Issue by greatnate58** [ultralytics/yolov5#5454]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I'm wondering why the number of batches for my validation set seems off during training. 

I'm training a small model with 707 validation images and a batch size of 34. That would work out to 21 batches for the validation set. The console output is showing only 11 batches over the val images. 

The number of batches for the training set seems correct. I have 2885 training images again at a 34 batch size. That works out to 85 batches, just as is shown in the console. 

![image](https://user-images.githubusercontent.com/83688349/139949084-54025535-fb13-4a6d-b308-fe4b9559c6d9.png)

Is train.py really only using half of my validation set? Or am I missing something?

**Comment by glenn-jocher** [comment]
@greatnate58 val operates correctly.

https://github.com/ultralytics/yolov5/blob/7476012a4de3a35c367fe30608be4e1ee89cc103/train.py#L352-L367

**Comment by greatnate58** [comment]
Thanks Glenn!
